README.md

```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/codeforamerica/click_that_hood.png?branch=master)](click_that_hood)
+[![Build Status][https://img.shields.io/travis/codeforamerica/click_that_hood/master.svg]][https://travis-ci.org/codeforamerica/click_that_hood]
```

Comment:
Thanks for asking.  Shields [provides](
https://github.com/badges/shields#solving-the-problem) consistent, resolution-independent informational images.